Core/GDCore/Project/Project.cpp

```diff
@@ -51,6 +51,7 @@ Project::Project()
     : name(_("Project")),
       version("1.0.0"),
       packageName("com.example.gamename"),
+      templateSlug(""),

```

Comment:
should this be NULL instead of "" ?

Reply:
Nope, in C++ a gd::String should always be a string, if you initialise it from a nullptr/NULL (i.e: 0) I guess very bad things can happen ;)

Reply:
Thanks :)

Reply:
Oops

Reply:
Some useful reading on why this would cause an issue: https://stackoverflow.com/questions/49058133/why-doesnt-stdstring-take-a-null-pointer